pkg/gadgets/trace/dns/tracer/bpf/dns-common.h

```diff
@@ -22,6 +22,7 @@ struct event_t {
 	// qr says if the dns message is a query (0), or a response (1)
 	unsigned char qr;
 	unsigned char pkt_type;
+	__u8 rcode;
```

Comment:
nit: Any reason not to use `unsigned char`? (I know, it's already messed up).

Reply:
no particular reason, I'll switch it to `unsigned char` for consistency with the above fields